src/aiida/cmdline/commands/cmd_computer.py

```diff
@@ -634,7 +634,7 @@ def _dry_run_callback(pks):
         if pks:
             echo.echo_report('The nodes with the following pks would be deleted: ' + ' '.join(map(str, pks)))
             echo.echo_warning(f'YOU ARE ABOUT TO DELETE {len(pks)} NODES! THIS CANNOT BE UNDONE!')
-        confirm = click.prompt('Shall I continue? [yes/N]', type=str) == 'yes'
+        confirm = click.confirm('Shall I continue?', default=False)
         if not confirm:
             raise click.Abort
         return not confirm
```

Comment:
May I take the chance to improve the message at line 642:
(if there are no nodes, this is a bit annoying saying "0 nodes marked for deletion")
```python
if node_pks: delete_nodes(node_pks, dry_run=dry_run or _dry_run_callback)
```